### ISSUE TITLE
Insufficient color contrast when active nav link is hovered

### ISSUE THREAD

**Issue by SatyamSetia** [AccessibleForAll/AccessibleWebDev#351]
**Describe the bug**
As per [1.4.3](https://webaim.org/articles/contrast/#:~:text=criteria%20in%20detail.-,1.4.3%20Contrast%20(Minimum),-This%20Level%20AA) success criteria in WCAG, the color contrast should be at-least 4.5:1 for the visible normal text. But in case of active nav link, when it is hovered, the color contrast is currently 1.56

**To Reproduce**
1. Hover over active nav link.
2. Try to inspect the color contrast in hovered state using developer tools

**Expected behavior**
Color contrast ratio should be at-least 1.56

**Screenshots**
<img width="1788" alt="Screenshot 2023-07-15 at 11 38 52 PM" src="https://github.com/AccessibleForAll/AccessibleWebDev/assets/17885747/415cacb2-5cd2-4b51-aea3-4841234dac98">

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Comment by SatyamSetia** [comment]
Hey, @EmmaDawsonDev
Just wanted to share my interest on working this bug fix if in-case you find it to be a relevant issue to be fixed.

My proposal for the fix -
In case there is specific design guideline for this case which makes it fulfil the contrast success criterion then lets go with that. Otherwise, lets make the text color to be black.

**Comment by EmmaDawsonDev** [comment]
Let's go ahead and make it black for now so that it passes. I'm going to work on the design a bit more in the meantime as I'm not 100% sure about it yet.

**Comment by SatyamSetia** [comment]
Acknowledged 👍
